src/spider/.clang-format

```diff
@@ -10,7 +10,7 @@ IncludeCategories:
   # Ex:
   # - Regex: "<(fmt|spdlog)"
   #   Priority: 3
-  - Regex: "^<(clp)"
+  - Regex: "^<(fmt|spdlog|boost|catch2|msgpack|mariadb)"
```

Comment:
- Alphabetize this list so that it's easy to see what libraries have already been added.
- What about abseil?